### DISCUSSION TITLE
Estimated cost incurred for Sunbird-ED installation on Azure?

### DISCUSSION THREAD

**Discussion by harryadwani** [Sunbird-Ed/Community#55] [Installation] [answered]
Hi, I would like to know the estimated cost incurred for Sunbird-ED installation on Azure. I understand that the cost would vary based on the region, modules and other parameters, but I would just like to know a rough estimate of out of the box installation, so that I can make a cloud service purchase accordingly. Thank you.

**Comment by sharathkashyap** [comment, marked as answer]
@gandham-santhosh  Can you help us with some details on cost

**Reply by gandham-santhosh** [reply]
@harryadwani Rough estimate of running Sunbird-ED on Azure cloud is $1000 (USD) per month considering infra is running 24x7

**Reply by parthlawate** [reply]
Can you also detail out if the assumption here is of a development instance ? As i understand the costs will vary as it scales up?

**Reply by smogili77** [reply]
hi @parthlawate 
We can talk if you have questions and i can share more details. You know my email and number already . Happy to share all the details you need
--SM

**Reply by parthlawate** [reply]
Sure ! 

@coolbung lets setup a call for a broader discussion ?  @smogili77 here my intent was to ensure when information is shared on the community more background information is given so it can be interpreted correctly :)

**Reply by smogili77** [reply]
Noted @parthlawate
